easybuild/easyblocks/w/wien2k.py

```diff
@@ -128,24 +134,30 @@ def configure_step(self):
         rplibs = ' '.join(rplibs)
 
         d = {
-             'FC': '%s %s' % (os.getenv('F90'), os.getenv('FFLAGS')),
-             'MPF': "%s %s" % (os.getenv('MPIF90'), os.getenv('FFLAGS')),
+             'FC': '%s' % os.getenv('F90'),
+             'FOPT': '%s' % os.getenv('OPTFLAGS'),
+             'MPF': '%s' % os.getenv('MPIF90'),
+             'FPOPT': '%s' % os.getenv('OPTFLAGS'),
              'CC': os.getenv('CC'),
              'LDFLAGS': '$(FOPT) %s ' % os.getenv('LDFLAGS'),
              'R_LIBS': rlibs,  # libraries for 'real' (not 'complex') binary
-             'RP_LIBS' : rplibs,  # libraries for 'real' parallel binary
+             'RP_LIBS': rplibs,  # libraries for 'real' parallel binary
              'MPIRUN': '',
             }
 
         for line in fileinput.input(self.cfgscript, inplace=1, backup='.orig'):
             # set config parameters
-            for (k,v) in d.items():
-                regexp = re.compile('^([a-z0-9]+):%s:.*' % k)
+            for (k, v) in d.items():
```

Comment:
@migueldiascosta While you're refactoring this, please avoid using single letter variables outside of list comprehensions, so please replace `k` with `key`, `v` with `val` and `d` with `vars`?

Reply:
will do